models/login_source.go

```diff
@@ -413,7 +413,7 @@ func LoginViaLDAP(user *User, login, password string, source *LoginSource, autoR
 	sr := source.Cfg.(*LDAPConfig).SearchEntry(login, password, source.Type == LoginDLDAP)
 	if sr == nil {
 		// User not in LDAP, do nothing
-		return nil, ErrUserNotExist{0, login, 0}
+		return nil, ErrUserNotExist{0, login, 0, -1}
 	}
```

Comment:
A default value ^^ correspoding to any value. 
I will review each case for determining. 
I was planning on keep -1 when we don't check the type of the user but for external sources this could be set to IndividualUser as it is supposed.